socorro/processor/breakpad_transform_rules.py

```diff
@@ -61,7 +61,9 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
 
 
 class ExternalProcessRule(Rule):
-
+    # FIXME(willkg): command_line and command_pathname are referenced in the
+    # uplifted versions in Processor2015. The rest of these config values have
+    # no effect on anything and are just here.
```

Comment:
I don't understand this comment. I read it to mean "These config values are not used anywhere", but I see `dump_field` being used in both `ExternalProcessRule` and `BreakpadStackwalkerRule2015`. Am I misunderstanding?

Reply:
It's so confusing. So `self.config.dump_field` in this class and subclasses actually refer to the value from `Processor2015`:

https://github.com/mozilla-services/socorro/pull/4350/files#diff-d49f4d6345a81fd5686209f8e3b86554L161

It doesn't refer to the `dump_field` option in this class or subclasses. This `dump_field` and the one in the subclasses don't have any effect on anything.

That's true because when these rules get instantiated here:

https://github.com/mozilla-services/socorro/pull/4350/files#diff-d49f4d6345a81fd5686209f8e3b86554L252

they get `config` for `Processor2015` passed to them which only contains keys/values defined in `Processor2015`.

I wanted to fix this but decided to toss in a FIXME for now and figure it out later.